bin/alluxio

```diff
@@ -230,6 +230,7 @@ function main {
     echo "Formatting Alluxio Master @ $(hostname -f)"
     CLASS="alluxio.cli.Format"
     PARAMETER="master"
+    ALLUXIO_SHELL_JAVA_OPTS+=" -Dalluxio.logger.type=Console"
```

Comment:
Is there a convention for if we should add the space before or after the option?